src/main/java/com/codicesoftware/plugins/hudson/commands/GetSelectorSpecCommand.java

```diff
@@ -19,7 +19,7 @@ public GetSelectorSpecCommand(String selectorPath) {
 
     public MaskedArgumentListBuilder getArguments() {
         MaskedArgumentListBuilder arguments = new MaskedArgumentListBuilder();
-        arguments.add("checkselectorsyntax");
+        arguments.add("css");
```

Comment:
This got my attention: in other instances you change abbreviated commands to use the full name, but this case is the other way around. Shouldn't we unify that?

Reply:
Yes. Correcting this.

Reply:
Ok, thanks for the update. We'll probably come up with some guidelines for this kind of stuff in the near future - e.g. always use full name commands instead of abbreviations.